gcfgenv.go

```diff
@@ -86,13 +86,23 @@ func readWithMapInto(r io.Reader, env map[string]string, prefix string, config i
 	return err
 }
 
+func gcfgTagOrStructName(field reflect.StructField) string {
```

Comment:
I'd suggest a function name that better conveys why you calling it, e.g.

```suggestion
func fieldToEnvVar(field reflect.StructField) string {
```

Reply:
Love the name, thanks! Great call!